lib/node_modules/@stdlib/blas/ext/base/dsortins/src/main.c

```diff
@@ -46,15 +60,15 @@ void c_dsortins( const int64_t N, const double order, double *X, const int64_t s
 	}
 	// For a positive stride, sorting in decreasing order is equivalent to providing a negative stride and sorting in increasing order, and, for a negative stride, sorting in decreasing order is equivalent to providing a positive stride and sorting in increasing order...
 	if ( order < 0.0 ) {
-		sx = -stride;
-	} else {
-		sx = stride;
+		strideX *= -1;
```

Comment:
You need to use `sx`, as in the original. There is no reason for the input `strideX` argument to not be a constant.